### PR TITLE
Flexure/helio test in run_pypeit

### PR DESCRIPTION
**Pull request by rcooke-ast**
Asserting the value of the flexure and helio during run_pypeit

The flexure assert is so that the difference is less than 0.1 pixels

### CODE REVIEW

**Review by profxj — APPROVED** [review]
Thanks for this.

File: pypeit/tests/test_runpypeit.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def test_run_pypeit():
     specObjs = specobjs.SpecObjs.from_fitsfile(spec1d_file)
 
     # Flexure
-    assert specObjs[0].FLEX_SHIFT_TOTAL != 0., 'Flexure not calculated!'
+    assert abs(specObjs[0].FLEX_SHIFT_TOTAL + 0.7792376147143614) < 0.1
```
Comment (kbwestfall) [inline pypeit/tests/test_runpypeit.py:142]: Is the number too specific?  I.e., could you just replace it with 0.78?  And/or can you use `np.isclose`?

**Review by kbwestfall — APPROVED** [review]
Approving, but with a comment for you to consider.

File: pypeit/tests/test_runpypeit.py (hunk 1 above)
Comment (rcooke-ast) [reply pypeit/tests/test_runpypeit.py]: Agreed - I've updated the number, and changed to `np.isclose` - thanks! I'll merge when travis passes

Comment (profxj) [reply pypeit/tests/test_runpypeit.py]: If you use `isclose`, you'll need to loosen the tolerance.
I do think that is a good idea.

Comment (rcooke-ast) [reply pypeit/tests/test_runpypeit.py]: OK - I've updated the code and the tests pass (I was doing two stupid things 😊)

**Comment by profxj** [comment]
The Travis failures are a new problem with `pytest` on their end.
Will see if I can figure out if we need to fix anything on our end to accomodate.

**Comment by profxj** [comment]
merging

**Pull request merged**